types/module/module.go

```diff
@@ -699,7 +699,7 @@ func (m Manager) RunMigrations(ctx context.Context, cfg Configurator, fromVM Ver
 				module1.InitGenesis(sdkCtx, c.cdc, module1.DefaultGenesis(c.cdc))
```

Comment:
Can we have a one-liner for the if condition, so that module1 variable is available only in the `if` branch.
Possibly rename them both to module as well instead of module1 / module2.